src/core/cgroup.c

```diff
@@ -181,6 +182,270 @@ void cgroup_context_init(CGroupContext *c) {
         };
 }
 
+int cgroup_context_copy(const CGroupContext *src, CGroupContext *dst) {
```

Comment:
in C usually when copying stuff target is first arg, source second. weird. but it is how it is, let's maybe stick to that rule here, i.e. swap the two args